packages/ella/package.json

```diff
@@ -60,15 +60,16 @@
     "rimraf": "^3.0.2",
     "tsup": "5.11.11",
     "typedoc": "^0.22.11",
-    "typescript": "^4.5.4"
+    "typescript": "^4.5.4",
+    "next": "12.3.0"
   },
   "peerDependencies": {
-    "react": "^16.12.0 || ^17.0.0"
+    "react": "^16.12.0 || ^17.0.0",
+    "next": "^12.0.0"
```

Comment:
remove ^

Reply:
Peer deps mein daalna padega prashant bhai, because we expect a range of versions. The peer dependencies are not installed for this package. In our main package (like web_apps) we maintain non caret version. Here it does not matter.